srcpkgs/cantarell-fonts/template

```diff
@@ -11,5 +11,5 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="OFL-1.1"
 homepage="https://wiki.gnome.org/Projects/CantarellFonts"
 changelog="https://gitlab.gnome.org/GNOME/cantarell-fonts/-/raw/master/NEWS"
-distfiles="${GNOME_SITE}/${pkgname}/${version}/${pkgname}-${version}.tar.xz"
-checksum=3d35db0ac03f9e6b0d5a53577591b714238985f4cfc31a0aa17f26cd74675e83
+distfiles="${GNOME_SITE}/${pkgname}/0.303/${pkgname}-${version}.tar.xz"
```

Comment:
https://github.com/archlinux/svntogit-packages/blob/packages/cantarell-fonts/trunk/PKGBUILD#L12

Uses `${version:0:5}` which looks like a great way to keep it future proof without hardcoding the version.

Reply:
Thanks, fixed :)